tests/test_freqtradebot.py

```diff
@@ -970,8 +970,10 @@ def test_handle_stoploss_on_exchange(mocker, default_conf, fee, caplog,
             'ask': 0.00001173,
             'last': 0.00001172
         }),
-        buy=MagicMock(return_value={'id': limit_buy_order['id']}),
-        sell=MagicMock(return_value={'id': limit_sell_order['id']}),
+        create_order=MagicMock(side_effect=[
+            (return_value={'id': limit_buy_order['id']}),
```

Comment:
Remove the return_value. An array as side_effect acts as return value unless the value returned is an exception.